cmd/cmd_test.go

```diff
@@ -1,8 +1,9 @@
 package cmd
 
 import (
-	"github.com/magiconair/properties/assert"
 	"testing"
+
+	"github.com/magiconair/properties/assert"
```

Comment:
Moving to stretchr/testify could further reduce deps :)

Reply:
Doesn't really help, but testify is more mainstream I guess 👍